pandas/core/arrays/string_.py

```diff
@@ -24,6 +21,7 @@
 
 from pandas.core.dtypes.base import (
     ExtensionDtype,
+    StorageExtensionDtype,
```

Comment:
is ExtensionType still neede?

Reply:
There's an `isinstance` check that uses it.